src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.MultiAgent.cs

```diff
@@ -199,6 +199,13 @@ internal void RequestUnpause(EasyRequest easy)
                 Queue(new IncomingRequest { Easy = easy, Type = IncomingRequestType.Unpause });
             }
 
+            /// <summary>Requests that the request associated with the easy operation be canceled..</summary>
```

Comment:
Thanks, Jeremy.  I'll address this in a subsequent PR/commit rather than restarting CI for it.